src/components/MewSheet/MewSheet.vue

```diff
@@ -1,31 +1,31 @@
-    <!--
-  =====================================================================================
-    Mew Sheet 
-    TODO: need to come back to this to add dark mode and figure out if we actually 
-    need this sheet
-    slot: content (used to place custom ui in the body)
-  =====================================================================================
-  -->
+<!--
+=====================================================================================
+  Mew Sheet 
+  TODO: need to come back to this to add dark mode and figure out if we actually 
+  need this sheet
+  slot: content (used to place custom ui in the body)
+=====================================================================================
+-->
 <template>
   <v-sheet
-    min-width="600"
-    width="100%"
-    color="white"
-    class="mew-sheet rounded-lg"
+    color="mewBg"
+    :class="[$vuetify.theme.dark ? 'box-shadow-dark' : 'box-shadow-light']"
   >
-    <slot name="content" />
+    <slot />
   </v-sheet>
 </template>
 
 <script>
-export default {
-};
+export default {};
 </script>
 
-
 <style lang="scss">
-.mew-sheet {
-   box-shadow: 0 12px 17px rgb(21 29 63 / 3%);
+.mew-component--white-sheet.box-shadow-light {
+  border-radius: 10px;
+  box-shadow: 0 0 15px var(--v-boxShadow-base) !important;
+}
+.mew-component--white-sheet.box-shadow-dark {
+  border-radius: 10px;
+  box-shadow: 0 0 15px #3c3c3c !important;
```

Comment:
also this color will have to be added in vuetifyTheme.js

Reply:
Fixed

Reply:
> also since you removed the named slot (content) , you will also have to remove it in MewSheet.stories.js - on line 32 or else it will break

fixed